ml-agents/mlagents/trainers/bc/trainer.py

```diff
@@ -31,7 +30,7 @@ def __init__(self, brain, trainer_parameters, training, load, seed, run_id):
         super(BCTrainer, self).__init__(brain, trainer_parameters, training, run_id)
         self.policy = BCPolicy(seed, brain, trainer_parameters, load)
         self.n_sequences = 1
-        self.cumulative_rewards = {}
+        self.cumulative_rewards = defaultdict(lambda: 0)
```

Comment:
```python
from typing import DefaultDict
...
self.cumulative_rewards: DefaultDict[str, float] = defaultdict(float)
```
(or DefaultDict[int, float] if agent id is an int)

Reply:
These will be removed in the logger PR to this branch